app/Http/Controllers/TransportController.php

```diff
@@ -23,6 +26,7 @@
 use GuzzleHttp\Client;
 use GuzzleHttp\Exception\GuzzleException;
 use GuzzleHttp\Exception\RequestException;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
 use Illuminate\Support\Collection;
 use Illuminate\Support\Facades\DB;
 use Illuminate\Support\Facades\Log;
```

Comment:
Remove unused imports

Reply:
You know that the ModelNotFoundException is used here? :D

Reply:
It was about MissingParametersExection which you've already removed.